src/model.rs

```diff
@@ -206,6 +206,12 @@ impl Model<ProblemCreated> {
             .set_cons_modifiable(cons, modifiable)
             .expect("Failed to set constraint modifiable");
     }
+    
+    /// Informs the SCIP instance that the objective value is always integral and returns the same `Model` instance.
+    pub fn set_obj_integral(mut self) -> Result<Self, Retcode> {
```

Comment:
I think I prefer it here to just return `Self` since calling it at the wrong stage is prevented at compile time, so calling this function should always work.

Reply:
Yes, you are absolutley right!